### PR TITLE
Adds DIO outfit to game and autodrobe

### PR DESCRIPTION
**Pull request by ghost**
So I figured I should learn how to use git and bawbaggings spent literally hours walking me through it.

This is the DIO set I made a week or so back. It's got nothing special about it at all, just a costume.

I've inevitably fucked the process up somewhere, but hopefully it's nothing critical.

ayy

### CODE REVIEW

**Comment by ghost** [comment]
"Mithrandalf wants to merge 1 commit into HippieStationCode:master from Mithrandalf:shitnipples"
Wryyyly good.
Although you need to modify the autodrobe refill for this to appear in the autodrobe round start.
https://github.com/HippieStationCode/HippieStation13/blob/07d647d436dc706cd4b317ba09b468e1fa7c1e6e/code/game/objects/items/weapons/vending_items.dm#L64

**Comment by ghost** [comment]
pls work

**Comment by ghost** [comment]
There's a ton wrong with this, I'll fix it tommorow.

**Comment by JohnGinnane** [comment]
When you've finished working on it can you test it on your machine to confirm you can get it from the autodrobe and that it doesn't break anything else?

**Comment by ghost** [comment]
Yeah that's what I've been doing fam, still cant figure out why the things dont appear worn.

**Comment by GeneralGoose** [comment]
@Mithrandalf that's because of the icon names and dm's you put them in (most of the time)

**Comment by GenericGoose** [comment]
nvm you fiksed it good

**Comment by Spacedong** [comment]
https://www.youtube.com/watch?v=uFqjIOWs8Cg

**Comment by ghost** [comment]
I'VE BEEN FUCKING THE CLOTHING VENDOR BECAUSE I THOUGHT IT WAS THE AUTODROBE.

FOR TWO FUCKING DAYS.

**Comment by ghost** [comment]
image
Should be working now. Do be sure to make sure I've not inadvertently fucked any of the other code though.

**Comment by Nebyab** [comment]
death to anime

**Comment by ghost** [comment]
askually it's called a manga nebyab

**Comment by Nebyab** [comment]
death to manga

**Comment by ghost** [comment]
It's time to stop.

**Comment by McHatters** [comment]
Yes pl0x

**Comment by dreamcrab** [comment]
is this busted still +666 if not busted

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -572,3 +572,19 @@
 	item_state = "telvis"
 	item_color = "telvis"
 	can_adjust = 0
+
+/obj/item/clothing/under/zootsuit
```
Comment (JohnGinnane) [inline code/modules/clothing/under/miscellaneous.dm:576]: This is new?

Comment (ghost) [inline code/modules/clothing/under/miscellaneous.dm:576]: Is this how I reply?

If so:

Nah, I think I might have added a line there or something tho.

**Comment by Spacedong** [comment]
KONO DIO DA

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (GenericGoose) [inline code/modules/clothing/under/miscellaneous.dm:576]: i believe zootsuit is the pimp suit from theatre vendor, it's a few months old

**Comment by GenericGoose** [comment]
is DIO dakimakura or whatever its kalled a thing

**Comment by Gandalf2k15** [comment]
Code is sound. @headmins please comment.

**Comment by Nebyab** [comment]
i already said death to anime

and death to mange

the fuck else do you need

**Comment by GenericGoose** [comment]
what a waste of sprites then

**Comment by ghost** [comment]
I guess I didn't actually say +1.
+1

**Comment by Spacedong** [comment]
merge this

**Comment by Gandalf2k15** [comment]
Merging as head input gained

**Pull request merged**